styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+    font-family: 'Poppins', sans-serif;
+    color: var(--color-white);
+    margin: 0;
+    background: var(--color-darkblue);
+    background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7)), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+form {
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+    max-width: 600px;
+    min-width: 300px;
+    background-color: rgba(27, 27, 50, 0.8);
+    color: #f5f6f7;
+    font-family: 'Poppins', sans-serif;
+    font-size: 18px
+}
+
+label {
+    display: block;
+    margin: 8px;
+}
+
+h1,
+p {
+    margin: 1em auto;
+    color: white;
+    text-align: center
+}
+
+fieldset {
+    border: none;
+    padding: 0.8rem 0;
+    border-bottom: none;
+}
+.top input {
+    display: block;
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+div {
+    padding-left: 25px;
+    padding-bottom: 25px;
+}
+.embed {
+    font-size: medium;
+}
+.dropdown{
+    margin-left: 3px;
+}
```

Comment:
- I think it's best to use `relative units` like `rem` or `em` for better site responsiveness rather than fixed units like `px`.